### ISSUE TITLE
(macOS) Scrollbar is hiding the dropdown arrow button in `Resolve conflicts before Rebase` dialog

### ISSUE THREAD

**Issue by SimonasB88** [desktop/desktop#17372]
### The problem

During the conflict resolving phase in the `Resolve conflicts before Rebase` dialog, the scrollbar is hiding the dropdown buttons, for conflicted files selection.

### Release version

Version 3.3.1 (arm64)

### Operating system

MacOS M1 Ventura 13.5.1

### Steps to reproduce the behavior

1. Just choose the branch in your repo
2. Rebase the branch, that causes conflicts
3. Wait for the Rebase dialog box

### Screenshots

<img width="100" alt="image" src="https://github.com/desktop/desktop/assets/29095235/7e6b90d6-7c70-4866-afc7-8bd36eafeb62">

**Comment by tidy-dev** [comment]
👋 @SimonasB88 Thank you for the report.

I initially could not reproduce it. But, then I could tell that the scrollbar in the screenshot is rendered differently than mine. Thus, I searched to see what kind of system scrollbar settings could be applied.  It appears mine is always rendering the scrollbar probably since I have an external mouse and set to "Automatically based on mouse or trackpad".  When I set it to, "When scrolling" I am able to reproduce. Thus, a workaround until we can add some padding is to set it to "Always". 

Shows the setting required to reproduce consistently:
![image](https://github.com/desktop/desktop/assets/75402236/a45823f1-7eb5-4dbb-bc1e-dbb2fc6899a0)
